ebs-janitor/main.go

```diff
@@ -43,6 +43,13 @@ func main() {
 		log.WithError(err).Error("failed initiate an AWS session")
 		return
 	}
+
+	dryrun := os.Getenv("dryrun")
+	if len(dryrun) > 0 && dryrun == "true" {
```

Comment:
I think this is bad practice here, as config is not part of the load config overall. You are trying to introduce a `dryrun` variable?